packages/api/src/providers/tenovi.ts

```diff
@@ -85,17 +82,20 @@ export class Tenovi extends Provider {
   async disconnectDevice(
     connectedUser: ConnectedUser,
     deviceId: string,
-    updateUser = true
+    updateUser: boolean,
+    extraParams: DAPIParams
   ): Promise<void> {
     const connectedDevices = connectedUser.providerMap?.tenovi?.connectedDeviceIds;
+    const { xTenoviApiKey, xTenoviClientName } = extraParams;
```

Comment:
Also required here, right?

Reply:
In the case with this function, these guys were made required upstream